versions/3.0.4.md

```diff
@@ -153,40 +190,40 @@ In order to preserve the ability to round-trip between YAML and JSON formats, YA
 
-#### <a name="resolvingImplicitConnections"></a>Resolving Implicit Connections
+#### Resolving Implicit Connections
 
 Several features of this specification require resolution of non-URI-based connections to some other part of the OpenAPI Description (OAD).
 
 These connections are unambiguously resolved in single-document OADs, but the resolution process in multi-document OADs is _implementation-defined_, within the constraints described in this section.
 In some cases, an unambiguous URI-based alternative is available, and OAD authors are RECOMMENDED to always use the alternative:
 
-Source | Target | Alternative
------- | ------ | -----------
-[Security Requirement Object](#securityRequirementObject) `{name}` |  [Security Scheme Object](#securitySchemeObject) name under the [Components Object](#componentsObject) | _n/a_
-[Discriminator Object](#discriminatorObject) `mapping` _(implicit, or explicit name syntax)_ | [Schema Object](#schemaObject) name under the Components Object | `mapping` _(explicit URI syntax)_
-[Operation Object](#operationObject) `tags` | [Tag Object](#tagObject) `name` (in the Components Object) | _n/a_
-[Link Object](#linkObject) `operationId` | [Path Item Object](#pathItemObject) `operationId` | `operationRef`
+| Source                                                                                       | Target                                                                                                | Alternative                       |
+| -------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- | --------------------------------- |
+| [Security Requirement Object](#security-requirement-object) `{name}`                           | [Security Scheme Object](#security-scheme-object) name under the [Components Object](#components-object) | _n/a_                             |
+| [Discriminator Object](#discriminator-object) `mapping` _(implicit, or explicit name syntax)_ | [Schema Object](#schema-object) name under the Components Object                                       | `mapping` _(explicit URI syntax)_ |
+| [Operation Object](#operation-object) `tags`                                                  | [Tag Object](#tag-object) `name` (in the Components Object)                                            | _n/a_                             |
+| [Link Object](#link-object) `operationId`                                                     | [Path Item Object](#path-item-object) `operationId`                                                     | `operationRef`                    |
 
```

Comment:
I have mixed feelings about this sort of table re-formatting.  On the one hand, it appeals to my sense of aesthetics.  On the other, it's very hard to get people to maintain it correctly.  I've gotten rather used to not bothering with consistent widths most of the time.  But I'm sure I bothered with it at least once, so... idk.

For me, it boils down to: "If a new contributor messes up this table spacing, will be block their PR until they fix it, and is that the sort of contributor experience we want to create?"

Reply:
Although if this is being done by `prettier` or some other script that we can set up to run as a pre-commit hook, then I'm 100% fine with it (and even the list item change, although I really do find `*` _much_ easier to spot when scanning text, as it is rarely used for anything other than lists, while `-` is used for many, many things and has less visual weight as weel).

Reply:
it is all automatic, a couple of tables had varying numbers of columns but otherwise this is all auto-fix and we can get pre-commit and/or CI to do it or at least check it